Translations/make_translation.py

```diff
@@ -1264,7 +1264,17 @@ def get_translation_sanity_checks_text(defs: dict) -> str:
 
 
 def get_version_suffix(ver) -> str:
+    # Check env var from push.yml first:
+    # - if it's pull request then use vX.YY + C.ID for version line as in *C*I with proper tag instead of merge tag for detached tree
+    try:
```

Comment:
You can possibly collapse this down using the .get(key,fallback) operator here if you wanted

Reply:
Very nice & tidy suggestion! [Seems working](https://github.com/Ralim/IronOS/actions/runs/5750316958/job/15586842858?pr=1772#step:8:1817).